src/appleseed.python/bindblenderprogressivetilecallback.cpp

```diff
@@ -46,6 +45,9 @@
 #include <cstddef>
 #include <vector>
 
+// OpenGL
+#include <glad/glad.h>
```

Comment:
We already have `foundation/platform/opengl.h`. Should be updated that file to use GLAD instead?

Reply:
I think this should be fine but we should test that it plays nice with Qt5's GL stuff

Reply:
Glad is only used for the blender progressive tile callback.  Qt is used for appleseed.studio so I can’t see where they would conflict

Reply:
Now that I think of it properly, maybe we don't even need `foundation/platform/opengl.h`, it looks like GLAD can fully replace it?

Reply:
There's one other place where `foundation/platform/opengl.h` is included: `appleseed.studio/mainwindow/rendering/lightpathswidget.cpp`. Would including GLAD there work? If so, we should get rid of our own OpenGL header.

Reply:
@jdent02 if it is used in `foundation/platform/opengl.h` then it would/should be able to be used by appleseed.studio as well. 

@dictoon Yea I think it probably can

Reply:
should I update  foundation/platform/opengl.h??

Reply:
I think instead you should try removing foundation/platform/opengl.h completely and replacing the other place where it exists with `glad/glad.h`

Reply:
even after removing foundation/platform/opengl.h
I keep getting this error: #error:  OpenGL header already included, remove this include, glad already

Reply:
appleseed.studio/mainwindow/rendering/lightpathswidget.cpp doesn't need foundation/platform/opengl.h. QGLWidget do the work by itself.